stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -62,13 +64,21 @@ class Point(NamedTuple):
     def dot(self, other): ...
     def angle(self, other): ...
     def mag(self): ...
+    @force_document
     def __add__(self, other): ...
+    @force_document
     def __sub__(self, other): ...
+    @force_document
     def __neg__(self): ...
+    @force_document
     def __mul__(self, other): ...
+    @force_document
     def __rmul__(self, other): ...
+    @force_document
     def __truediv__(self, other): ...
+    @force_document
     def __floordiv__(self, other): ...
+    @force_document
```

Comment:
`@force_document` and `@force_nodocument` do nothing in the stub file, because these decorators return the same type `_T` that is passed to them as input. I suggest deleting them, unless you have a specific reason to keep them.

Reply:
Sure, I'll remove it.